opendps/hw.c

```diff
@@ -696,9 +719,23 @@ void BUTTON_M1_isr(void)
     static bool falling = true;
     exti_reset_request(BUTTON_M1_EXTI);
     if (falling) {
+        m1_pressed = true;
+        if (m2_pressed)
+            m1_and_m2_pressed = true;
         exti_set_trigger(BUTTON_M1_EXTI, EXTI_TRIGGER_RISING);
     } else {
-        event_put(event_button_m1, press_short);
+        m1_pressed = false;
+
+        if (!m2_pressed) {
+            if (m1_and_m2_pressed) {
+                m1_and_m2_pressed = false;
+                event_put(event_buttom_m1_and_m2, press_short);
+            }
```

Comment:
May I be picky about braces? ;)

Reply:
How do you prefer the braces? Allman style?

Reply:
I am actually a bit of an K&R guy.